### ISSUE TITLE
Create Search Term For Works That Are Subjects Of Other Works

### ISSUE THREAD

**Issue by clarkepeterf** [project-lux/lux-marklogic#66]
**Problem Description:**
Frontend/middle tier want to display Works that are the subject of other works

**Expected Behavior/Solution:**
Create a field and search term that allow to search for works that are the subject of other works

**Requirements:**
@azaroth42  @kkdavis14 do we want to search in both directions? E.g.:
a. Works that are about this work
b. Works that this work is about

In that case we will need the pipeline to create a new triple 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Questions
  - [ ] From above:  Do we want to search in both directions? E.g.:
    a. Works that are about this work
    b. Works that this work is about
~- List of questions for discussions. Answers should be documented within the issue.~

**UAT/LUX Examples:**

~~- Example endpoints from LUX to use for development and testing, if applicable.~~

**Dependencies/Blocks:**

~~- Blocked By: Issues that are blocking the completion of the current issue.~~
- Blocking: https://github.com/project-lux/lux-middletier/issues/28

**Related Github Issues:**
- https://github.com/project-lux/lux-frontend/issues/62

**Related links:**

- https://www.bugherd.com/projects/284041/tasks/1792

**Wireframe/Mockup:**
~~Place wireframe/mockup for the proposed solution at end of ticket.~~

**Comment by kkdavis14** [comment]
FYI this also includes Objects--do you have a separate ticket for that? i.e. first sentence should be frontend wants to displayed Works and Objects that are the subject of other Works.

**Comment by kkdavis14** [comment]
for Question b: that's in the data.

**Comment by clarkepeterf** [comment]
@prowns  #agenda

**Comment by brent-hartwig** [comment]
@clarkepeterf, any interest in doing this alongside #74?

cc: @roamye, @kamerynB

**Comment by kamerynB** [comment]
@brent-hartwig I worked on 74 which has an [open PR](https://github.com/project-lux/lux-marklogic/pull/268). What would be needed for this?

**Comment by brent-hartwig** [comment]
Ah, the status is still Forming and I saw the `index change` label.  If you've already got that covered (or an index change is not needed), you can ignore me :)

**Comment by brent-hartwig** [comment]
@kamerynB, maybe nothing?  I didn't realize they could be duplicates of each other.  If they need different search terms, I'd expect whatever you did for #74 needed here as well.

**Comment by roamye** [comment]
> FYI this also includes Objects--do you have a separate ticket for that? i.e. first sentence should be frontend wants to displayed Works and Objects that are the subject of other Works.

@kkdavis14 - as far as I know, there was no other ticket created for the Objects. Do we need all new tickets for those? (FE/MT/ML)

**Comment by kkdavis14** [comment]
@roamye it looks like this ticket covers Objects. Title is "create search term for Objects or Works that Are Subjects of Other Works". Ticket description did not get updated, but as long as this is for both Objs & Works, all good.

**Comment by kamerynB** [comment]
I am proposing this be closed as it is a dupe of ML #74.
